impl/src/main/java/org/jboss/weld/contexts/CreationalContextImpl.java

```diff
@@ -42,6 +44,7 @@
  * @author Pete Muir
  * @author Ales Justin
  * @author Jozef Hartinger
+ * @author <a href="https://about.me/lairdnelson" target="_parent">Laird Nelson</a>
```

Comment:
Please don't add yourself; we should probably remove these sections as they are outdated in many place anyway.
Plus git and GH are good enough tools to determine authors and contributors. In fact they are far better as they can show you exactly who authored any part of the code.

Reply:
Done